src/assets/dresses/suspension_harness/suspension_harness.asset.ts

```diff
@@ -0,0 +1,78 @@
+DefineAsset({
+	name: 'Harness Jute Ropes',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		rope: {
+			name: 'Rope',
+			default: '#D7AC4D',
+		},
+	},
+	// size:200, y:400, centered
+	preview: 'preview.png',
+	attributes: {
+		provides: [
+			'Restraint',
+		],
+	},
```

Comment:
[TODO for me and Claudia] This needs some category to be put into. Preferably a new "body restraints" attribute.